src/modules/storage/LedgerStorage.ts

```diff
@@ -1560,9 +1647,12 @@ export class LedgerStorage extends Storages
 
         let sql_receiver =
             `SELECT
-                O.address,
+                O.output_index,
                 O.amount,
-                O.utxo_key as utxo
+				O.lock_type,
+				O.lock_bytes as bytes,
```

Comment:
```suggestion
		O.lock_type,
		O.lock_bytes as bytes,
```

Reply:
resolved

Reply:
Not resolved yet

Reply:
![codeformat](https://user-images.githubusercontent.com/78407888/117402659-42d4f900-af20-11eb-99f4-06e1935928da.png)

Reply:
i resolved this.